docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "2"
+services:
+  rabbit:
+    container_name: rabbitmq
+    image: rabbitmq:3.6-management
+    ports:
+        - "15673:15672" # Exposing RabbitMQ web management on different port for convenience
+    restart: always
+
+  postgres:
+    container_name: postgres_db
+    image: postgres
+    ports:
+        - "5433:5432" # Exposing Postgres on different port for convenience
+    environment:
+        POSTGRES_DB: "orders"
+        POSTGRES_PASSWORD: "password"
+        POSTGRES_USER: "postgres"
+    restart: always
+
+  orders:
+    container_name: orders
+    image: nameko/nameko-example-orders:latest
+    depends_on:
+      - rabbit
+      - postgres
+    ports:
+        - "8001:8000"
+    links:
+        - "rabbit:rabbit"
+        - "postgres:postgres"
+    environment:
+        DB_PASSWORD: "password"
+        DB_USER: "postgres"
+        DB_HOST: "postgres"
+        DB_NAME: "orders"
+        RABBIT_PASSWORD: "guest"
+        RABBIT_USER: "guest"
+        RABBIT_HOST: "rabbit"
+        RABBIT_PORT: "15672"
```

Comment:
Actually there is a problem here: `RABBIT_PORT` is used to construct both the `AMQP_URI` in `config.yaml` _and_ in `run.sh::is_ready()`. You need `5672` for one and `15672` for the other.